packages/story-editor/src/components/panels/design/link/karma/link.karma.js

```diff
@@ -78,7 +78,8 @@ describe('Link Panel', () => {
 
   describe('CUJ: Creator Can Add A Link: Apply a link to any element', () => {
     beforeEach(async () => {
-      await fixture.events.click(fixture.editor.library.textAdd);
+      await fixture.editor.library.textTab.click();
+      await fixture.events.click(fixture.editor.library.text.preset('Title 1'));
```

Comment:
Hmm, any clue why `textAdd` doesn't work?

Reply:
It looks like it's so small in the test that the previous coordinates for hovering didn't always work.